tests/foreman/ui/test_computeresource_ec2.py

```diff
@@ -45,7 +45,6 @@ def module_ec2_settings():
 
 @pytest.mark.tier2
 @pytest.mark.skip_if_not_set('http_proxy')
-@pytest.mark.skip_if_open("BZ:2032530")
```

Comment:
How does merely removing the conditional skip fix the issue? The BZ got closed so it's ok to remove but there is a workaround needed.

Reply:
My local setup was not working so I was trying to find the actual failure with the PRT. There is an issue with the subnet selection so fixing that issue in this PR.